test/e2e/src/common/kubernetes_configmap_utility.py

```diff
@@ -0,0 +1,8 @@
+import pytest
```

Comment:
Is a separate file needed for this? It's pretty short
(I'm just posing the question so you think about it. Feel free to resolve this comment immediately)

Reply:
> @daweim0 , as I mentioned in the scrum, we will have to add more functions as we add more tests. Right now, source files are separated based on the resource type.